kuma/wiki/tests/test_content.py

```diff
@@ -964,7 +964,7 @@ def test_editor_safety_filter(self):
             <svg><circle onload=confirm(3)>
             <h1 class="header1">Header One</h1>
             <p>test</p>
-            <section>
+            <section onclick="alert('hacked!')">
```

Comment:
I'm surprised this was needed - ``filterEditorSafety`` is supposed to remove it, which I think it is whole purpose of this test.

Reply:
On second look, you're right, it doesn't add much new here, other than adding a different flavor of `on*` attribute to the one already there (`onload=confirm(3)`). I must have been thinking in terms of how this contrasts with `bleach.clean`, which would escape the element with the `onload=confirm(3)` but strip the `onclick`. If you don't mind, I think I'll leave it though.

Reply:
OK, we can leave it, but this whole test should be removed when we remove ``filterEditorSafety``.